app/classifier/tasks/drawing/summary.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Markdown } from 'markdownz';
+import SummaryAnswer from './summary-answer';
+
+class DrawingSummary extends React.Component {
+  static propTypes = {
```

Comment:
I would specify propTypes as `DrawingSummary.propTypes = {...}`. Is there a reason to use the static property instead?

Reply:
`static` is ES6 syntax for declaring static methods and properties. I don't think there's any difference between using the ES6 syntax or declaring static properties using `ClassName.property`. We added `static` to babel during the React 16 upgrade (535c0125b031ef8ef73ead6df1ffc404d8d4f3fc) so it's been used in some of the recent ES6 components. I think babel converts this to the old-style `DrawingSummary.propTypes` anyway.